internal/pkg/handlers/appInsights.go

```diff
@@ -0,0 +1,175 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/lawrencegripper/azbrowse/internal/pkg/style"
+	"github.com/lawrencegripper/azbrowse/pkg/armclient"
+)
+
+type analyticsItem struct {
+	Content      string `json:"Content"`
```

Comment:
Nice - This is an interesting option for types that aren't going to be re-used by other handlers. Maybe a nice approach going forward, I could pull the ones I added for metrics into the handlers.